.circleci/docker/Dockerfile.ubuntu1904

```diff
@@ -39,29 +44,24 @@ RUN set -ex; \
 	apt-get install -qy python-pip python-sphinx; \
 	update-alternatives --install /usr/bin/llvm-symbolizer llvm-symbolizer /usr/bin/llvm-symbolizer-8 1; \
 	pip install codecov; \
-	rm -rf /var/lib/apt/lists/*
-
-# Aleth for end-to-end tests
```

Comment:
Minor comment, since the single run command is now much bigger, might be good to retain comments denoting beginning of a dependency for readability, what I mean is something like

```
# Aleth for end-to-end tests
<aleth_install_cmds>
# Z3
<z3_install_cmds>
...
and so on
```

Reply:
I am a bit confused, every section _does_ already have a comment at the beginning?

Reply:
Ah, every section except this one because I movied the comments with the `ARGS`

Reply:
fixed & pushed

Reply:
ah, you are right, I don't know why I extrapolated to other sections. Thanks for the update.